### ISSUE TITLE
Check if simulator exactable is 32bit?

### ISSUE THREAD

**Issue by themperek** [cocotb/cocotb#2911]
We could check if the simulator executable is 32 or 64 bit and rise an error in case of incompatibility.

Like this? https://stackoverflow.com/questions/61999530/checking-if-an-executable-is-32-bit-or-64-bit-with-a-python3-script-on-windows-l

**Comment by imphil** [comment]
Good idea! That would need to be implemented in the Makefile and/or in cocotb runner.
